bookkeeper-server/src/main/java/org/apache/bookkeeper/client/api/CreateBuilder.java

```diff
@@ -91,6 +93,26 @@
      */
     CreateBuilder withDigestType(DigestType digestType);
 
+    /**
+     * Set write flags. Write wlags specify the behaviour of writes
+     *
+     * @param writeFlags the flags
+     *
+     * @return the builder itself
+     */
+    CreateBuilder withWriteFlags(EnumSet<WriteFlag> writeFlags);
```

Comment:
I think it'd be better to have withDeferredSync. i.e. A builder method for each write flag, and not expose the write flag enum to the client in any way.

Reply:
I am not sure though. Coming from a storage perspective, `withWriteFlags` is more close to a storage behavior. We have many rounds discussing how the API looks like towards an extensible interface for WriteFlags, ReadFlags and StorageHints. I would prefer stick to what we have agreed on.

Reply:
ok